### PR TITLE
OBPIH-2547 Disable pack page for locations withoug packing activity

### PR DESCRIPTION
**Pull request by DominikaHatala**
This is v0.8.12 ticket

### CODE REVIEW

File: src/groovy/org/pih/warehouse/core/ActivityCode.groovy [hunk 1]
```diff
@@ -37,8 +37,10 @@ enum ActivityCode {
     MODIFY_PICKLIST_ITEM('MODIFY_PICKLIST_ITEM'),
 
     // Notifications
-    ENABLE_NOTIFICATIONS('ENABLE_NOTIFICATIONS')
+    ENABLE_NOTIFICATIONS('ENABLE_NOTIFICATIONS'),
 
+    // Packing
+    PACKING('PACKING')
```
Comment (jmiranda) [inline src/groovy/org/pih/warehouse/core/ActivityCode.groovy]: Change this to PACK_SHIPMENT to be consistent with the VERB_NOUN convention.

File: grails-app/i18n/messages.properties [hunk 2]
```diff
@@ -640,6 +640,7 @@ enum.ActivityCode.PICK_STOCK=Pick stock
 enum.ActivityCode.PUTAWAY_STOCK=Putaway stock
 enum.ActivityCode.CROSS_DOCKING=Cross Docking
 enum.ActivityCode.ENABLE_NOTIFICATIONS=Enable Notifications
+enum.ActivityCode.PACKING=Packing
```
Comment (jmiranda) [inline grails-app/i18n/messages.properties]: Again, change this to be consistent
```
enum.ActivityCode.PACK_SHIPMENT=Pack Shipment
```

File: grails-app/conf/BootStrap.groovy [hunk 3]
```diff
@@ -148,7 +149,8 @@ class BootStrap {
                     parentLocation       : location.parentLocation,
                     locationType         : location.locationType,
                     sortOrder            : location.sortOrder,
-                    hasBinLocationSupport: location.hasBinLocationSupport()
+                    hasBinLocationSupport: location.hasBinLocationSupport(),
+                    hasPackingSupport    : location.supports(ActivityCode.PACKING)
```
Comment (jmiranda) [inline grails-app/conf/BootStrap.groovy]:
```
hasPackShipmentSupport : location.supports(ActivityCode.PACK_SHIPMENT)
```

**Comment by DominikaHatala** [comment]
@jmiranda changes applied

**Comment by awalkowiak** [comment]
@DominikaHatala  conflicts

**Pull request merged**